docusaurus/docs/guides/index.md

```diff
@@ -3,13 +3,15 @@ title: Overview
 sidebar_position: 10
 ---
 
-## What is a Guide?
-
-A guide is like a recipe. It's a type of documentation that is more advanced and solution-focused than other types. A guide is prescriptive and procedural. For example, "How to deploy a Ziti controller with Helm."
+This section contains operational guidance for specific tasks such as deploying a controller with Helm or as a Linux system service, backing up the controller's database, and migrating controller or router configurations.
 
 ## Add or Request New Guides
 
 There's lots of room in here for new guides! Want guidance before you submit? Want someone else to submit? No problem. Follow the link to create a GitHub issue, label it "how-to", and tell us what you are working on. 
 
 * [Request a new guide or find guide requests](https://github.com/openziti/ziti-doc/issues?q=is%3Aissue+is%3Aopen+label%3Ahow-to+)
 * [Where in GitHub to add or edit a guide](https://github.com/openziti/ziti-doc/tree/main/docusaurus/docs/guides)
+
+## What is a Guide?
```

Comment:
I'd like to see this where it used to be and the other text removed or incorporated into this